### DISCUSSION TITLE
How to define the num of workflow

### DISCUSSION THREAD

**Discussion by yanghaogn** [Netflix/conductor#2806] [Q&A] [answered]
As in [https://github.com/Netflix/conductor/discussions/2299](https://github.com/Netflix/conductor/discussions/2299), we may run 100 million workflows. If 3 workflowId use one workflow name, is the num 1 or 3 ?

**Comment by v1r3n** [comment, marked as answer]
this refers to the no. of workflow executions. so if you have 1 workflow name and 100M executions, that is 100M workflow executions per day.
